sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Breaking Changes
 
+- Renamed `keyvault_certificates.hpp` to `certificates.hpp`.
```

Comment:
What motivated this? For keyvault keys, our simplified header is called keyvault_keys.hpp.

Reply:
If keyvault include files are in `azure/keyvault/certificates/`, then the all-inclusive file is `azure/keyvault/certificates.hpp`.